angular-ngrx-scss/src/app/state/profile/profile.state.ts

```diff
@@ -86,26 +86,7 @@ export interface UserRepo {
   name: string;
   full_name: string;
   private: boolean;
-  owner: {
-    login: string;
-    id: number;
-    node_id: string;
-    avatar_url: string;
-    gravatar_id: string;
-    url: string;
-    html_url: string;
-    followers_url: string;
-    following_url: string;
-    gists_url: string;
-    starred_url: string;
-    subscriptions_url: string;
-    organizations_url: string;
-    repos_url: string;
-    events_url: string;
-    received_events_url: string;
-    type: string;
-    site_admin: boolean;
-  };
+  owner: UserApiResponse;
```

Comment:
I don't think we should store the information in the state in the exact shape as data returned from API. Or at least keep 2 separate interfaces for API and State data.

Reply:
We have a ticket created already to clean up and improve the interfaces. I'll add the state files to be checked and cleaned to that as well.